internal/pkg/manifest/env.go

```diff
@@ -521,13 +520,24 @@ type RestrictiveIngress struct {
 	CDNIngress *bool `yaml:"cdn"`
 }
 
+// RelaxedIngress represents ingress field which relaxes default
+// behavior of not allowing any public ingress.
```

Comment:
nit:
```suggestion
// RelaxedIngress contains ingress fields that relax the default ingress.
```

Reply:
If we can let's not reuse the word "relax" or "restrict" in the comments to add some more clarity:
```suggestion
// RelaxedIngress contains ingress configuration to add to a security group.
```